Makefile

```diff
@@ -1,3 +1,4 @@
+hid-uclogic-objs := hid-uclogic-rdescs.o
```

Comment:
Could you please move this line below the following one? This way we'll still have the list of modules at the top and their details will follow.